lib/src/ast/sass/expression/binary_operation.dart

```diff
@@ -88,51 +88,48 @@ class BinaryOperationExpression implements Expression {
 /// A binary operator constant.
 ///
 /// {@category AST}
-@sealed
-class BinaryOperator {
+enum BinaryOperator {
   /// The Microsoft equals operator, `=`.
```

Comment:
Can you run Dartdoc and make sure the option documentation still shows up properly?

Reply:
Ah it seems the "new way" is `[MyClass.new]` or `[MyClass.myNamedConstructorOrFactory]`

Reply:
browsing through these, they look correct, they are now nested under an "Enums" subsection but otherwise symbols and everything looks better actually because the "values" appear more upfront compared to static properties, although I'm not sure if you meant Operator instead of Option 🤔